docs/source/nodes_and_pipelines/namespaces.md

```diff
@@ -0,0 +1,221 @@
+# Reuse pipelines with namespaces
+
+## How to reuse your pipelines
+
+If you want to create a new pipeline that performs similar tasks with different inputs/outputs/parameters as your existing pipeline (e.g., `data_science`), you can use the same `pipeline()` creation function as described in [How to structure your pipeline creation](modular_pipelines.md#how-to-structure-your-pipeline-creation). This function allows you to overwrite inputs, outputs, and parameters. Your new pipeline creation code should look like this:
+
+```python
+def create_new_pipeline(**kwargs) -> Pipeline:
+    return pipeline(
+    [data_science], # Name of an existing pipeline
```

Comment:
Where does `data_science` come from in this example?

Reply:
I refer to a `data_science` pipeline from the Spaceflights starter tutorial. In this example, it's clearer why a user might want to reuse parts of the pipeline in a business case, compared to an example using medians and means. I've included links to the tutorial.